### ISSUE TITLE
patientSelectNative option not working

### ISSUE THREAD

**Issue by YogendraShelke** [smart-on-fhir/Swift-SMART#7]
patientSelectNative option as SMARTAuthGranularity not working for epic and careEvolution sandbox

**Comment by p2** [comment]
Yes, this is a known issue. The "native" selector asks for a list of all patients, which I know Epic does not provide unless at least basic search criteria are provided. Hence 0 patients are returned.
